tests/test_concordance_cc.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+import audmetric
+
+
+def expected_ccc(truth, prediction):
+    r"""Expecte Concordance Correlation Coefficient.
+
+    This is a direct implementation of its math equation.
+
+    If only a single sample is given,
+    it should return NaN.
+
+    """
+    prediction = np.array(list(prediction))
+    truth = np.array(list(truth))
+
+    if len(prediction) < 2:
```

Comment:
Do we actually need those special cases where we return `np.NaN` or can we simplify the function now?

Reply:
Sorry, forgot to remove this. We don't need this and it is now removed.